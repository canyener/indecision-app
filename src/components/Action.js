import React from 'react'

const Action = (props) => {
  return (
    <div>
      <button 
        onClick={props.handlePick}
        disabled={!props.hasOptions}
      >
        What should I do?
      </button>
    </div>
  )
}

export default Action