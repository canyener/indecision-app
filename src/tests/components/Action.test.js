import React from 'react'
import { shallow } from 'enzyme'

import Action from '../../components/Action'

test('Should render Action component correctly', () => {
  const wrapper = shallow(
    <Action 
      handlePick={() => {}} 
      hasOptions={true} 
    />
  )

  expect(wrapper).toMatchSnapshot()
})
