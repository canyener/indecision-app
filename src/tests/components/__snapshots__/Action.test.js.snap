// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Action component correctly 1`] = `
<div>
  <button
    className="big-button"
    disabled={false}
    onClick={[Function]}
  >
    What should I do?
  </button>
</div>
`;
